expan/core/experiment.py

```diff
@@ -253,10 +249,11 @@ def _is_valid_for_analysis(self, data, test):
         :return True if data is valid for analysis and False if not
         :rtype: bool 
         """
-        if len(data[data[test.variants.variant_column_name] == test.variants.control_name].dropna()) <= 1:
+        column_name = test.variants.variant_column_name
+        if len(data.loc[data[column_name] == test.variants.control_name, column_name].dropna()) <= 1:
             logger.warning("Control group only contains 1 or 0 entities.")
             return False
-        if len(data[data[test.variants.variant_column_name] == test.variants.treatment_name].dropna()) <= 1:
+        if len(data.loc[data[column_name] == test.variants.treatment_name, column_name].dropna()) <= 1:
```

Comment:
do you still need `dropna()`? I didn't try it myself, but I doubt that condition returns True for NAs.
`data[column_name] == test.variants.treatment_name`

Reply:
no we don't need it:)